x/liquidstaking/spec/02_state.md

```diff
@@ -106,19 +106,23 @@ This will be consumed at **Handle Queued Withdraw Insurance Requests** when Epoc
 
 - **reward module account’s native token(e.g. `acanto`) balance**
 - **sum of all chunk balance**
```

Comment:
I think, pairing chunk(unbonded) is omitted on netAmount

Reply:
I think pairing chunk's balance is included in `sum of all chunk balance`.